res/controllers/hid-controller-api.d.ts

```diff
@@ -0,0 +1,61 @@
+
+/** HidControllerJSProxy */
+
+declare namespace controller {
+    /**
+     * Sends HID OutputReport with hard coded ReportID 0 to HID device
+     *        This function only works with HID devices, which don't use ReportIDs
+     *
+     * @param dataList Data to send as list of bytes
+     * @param length This optional argument is no longer evaluated, and only here for backwards compatibility [default = 0]
+     */
+    function send(dataList: number[], length?: number): void;
+
+    /**
+     * Sends HID OutputReport to HID device
+     *
+     * @param dataList Data to send as list of bytes
+     * @param length Unused but mandatory argument
```

Comment:
Otherwise the statement sounds somewhat contradictory.
```suggestion
     * @param length Unused but mandatory argument for backwards compatibility
```